.github/hookdoc-tmpl/README.md

```diff
@@ -10,4 +10,39 @@ Version 2.0 of Distributor contains breaking changes. Please review the <a href=
 
 To report an issue with Distributor or contribute back to the project, please visit the [GitHub repository](https://github.com/10up/distributor/).
 
+## Developers
+
+### Running Locally
+
+If you are compiling Distributor locally, note that the recommended version of Node.js is version 16.x. The minimum version required is Node.js 8.10.
+
+An `.nvmrc` file is included in the plugin repository. It's recommended you install [fnm (fast node manager)](https://github.com/Schniz/fnm/), [nvm (node version manager)](https://github.com/nvm-sh/nvm) or similar when developing locally.
+
+### Testing
+
+The plugin contains a standard test suite compatible with PHPUnit. If you want to test across multiple PHP versions, a [Dockunit](https://github.com/dockunit/dockunit) file is included.
+
+### Debugging
+
+You can define a constant `DISTRIBUTOR_DEBUG` to `true` to increase the ease of debugging in Distributor. This will make all remote requests blocking and expose the subscription post type.
+
+Enabling this will also provide more debugging information in your error log for image side loading issues. The specific logging method may change in the future.
+
+### Application Passwords and WordPress 5.6
```

Comment:
We support a minimum WordPress `5.7`. Can we update this?

Reply:
I removed the "and WordPress 5.6" text since that isn't relevant anymore.